services/ui-src/src/components/modals/AddEditEntityModal.test.tsx

```diff
@@ -100,12 +96,14 @@ describe("Test AddEditEntityModal", () => {
 
 describe("Test AddEditEntityModal functionality", () => {
   afterEach(() => {
+    mockUpdateCallPayload.fieldData.accessMeasures.splice(-1);
```

Comment:
Suggestion: What if we did something like this, so it's more clear what we're up to.
```suggestion
    // reset payload to baseline with only mockEntity
    mockUpdateCallPayload.fieldData.accessMeasures = [mockEntity];
```